docs/docs/configuration/group.md

```diff
@@ -0,0 +1,85 @@
+---
+id: allowed_groups
+title: Allowing group based access
+---
+
+Limiting access to a resource based on the users group membership gives you a finer level of access control.  
+You will need an oauth provider that supports adding the `group` claim in the `id_token`.
+
+Configured providers are :
+
+- [Azure](#azure-auth-provider)
+
+### Azure Auth Provider
+
+In azure, you can configure additional claims for a service principal in general as described here
+[https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-optional-claims](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-optional-claims).
```

Comment:
Might be better to have an inline link, perhaps we can reword it to flow more naturally to allow a link
```suggestion
Before you can use groups with the Azure provider, you must [configure additional claims](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-optional-claims) for a service principal using the Azure portal.
```
WDYT?